app/javascript/components/subnet-form/subnet-form.schema.js

```diff
@@ -30,7 +30,7 @@ const createSchema = (edit, fields = [], loadSchema, emptySchema) => ({
           showError = true;
         }
       },
-      loadOptions: () => networkManagers,
+      loadOptions: () => networkManagers(),
```

Comment:
This change is not needed

Reply:
I tried it without the () and it wasn't working, the form would just not load. I can try again but I have to fix my local environment.

Reply:
I think you do need it because the loadOptions function needs to execute the function (as opposed to returning the function itself)

Reply:
Actually @kavyanekkalapu I see what you mean elsewhere we don't execute the function, but just reference it

e.g. https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/javascript/components/cloud-tenant-form/cloud-tenant-form.schema.js#L35

Reply:
Oh no wait - notice in the example I shared it passes the function directly instead of wrapping it.. That is, compare:

// cloud-tenant-form.schema.js
      loadOptions: loadProviders,
// vs
      loadOptions: () => networkManagers(),

Reply:
@GilbertCherrie So I think this line should be

```suggestion
      loadOptions: networkManagers,
```